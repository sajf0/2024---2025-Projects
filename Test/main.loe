\addvspace {10\p@ }
\contentsline {theorem}{\ifthmt@listswap Theorem~(1 - 1)\else \numberline {(1 - 1)}Theorem\fi \thmtformatoptarg {Lagrange}}{1}{}%
