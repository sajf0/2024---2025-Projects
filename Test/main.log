This is XeTeX, Version 3.141592653-2.6-0.999996 (TeX Live 2024/Arch Linux) (preloaded format=xelatex 2024.11.12)  20 MAR 2025 15:30
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**main.tex
(./main.tex
LaTeX2e <2023-11-01> patch level 1
L3 programming layer <2024-02-20>
(/usr/share/texmf-dist/tex/latex/extsizes/extreport.cls
Document Class: extreport 1996/10/08 v1.0 Non Standard LaTeX document class
(/usr/share/texmf-dist/tex/latex/extsizes/size14.clo
File: size14.clo 1999/11/11 v1.4a NON-Standard LaTeX file (size option)
)
(/usr/share/texmf-dist/tex/latex/base/exscale.sty
Package: exscale 2018/09/24 v2.1i Standard LaTeX package exscale
LaTeX Font Info:    Redeclaring symbol font `largesymbols' on input line 57.
LaTeX Font Info:    Overwriting symbol font `largesymbols' in version `normal'
(Font)                  OMX/cmex/m/n --> OMX/cmex/m/n on input line 57.
LaTeX Font Info:    Overwriting symbol font `largesymbols' in version `bold'
(Font)                  OMX/cmex/m/n --> OMX/cmex/m/n on input line 57.
\big@size=\dimen140
)
\c@part=\count184
\c@chapter=\count185
\c@section=\count186
\c@subsection=\count187
\c@subsubsection=\count188
\c@paragraph=\count189
\c@subparagraph=\count190
\c@figure=\count191
\c@table=\count192
\abovecaptionskip=\skip48
\belowcaptionskip=\skip49
\bibindent=\dimen141
)
(/usr/share/texmf-dist/tex/latex/geometry/geometry.sty
Package: geometry 2020/01/02 v5.9 Page Geometry

(/usr/share/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2022/05/29 v1.15 key=value parser (DPC)
\KV@toks@=\toks17
)
(/usr/share/texmf-dist/tex/generic/iftex/ifvtex.sty
Package: ifvtex 2019/10/25 v1.7 ifvtex legacy package. Use iftex instead.

(/usr/share/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2022/02/03 v1.0f TeX engine tests
))
\Gm@cnth=\count193
\Gm@cntv=\count194
\c@Gm@tempcnt=\count195
\Gm@bindingoffset=\dimen142
\Gm@wd@mp=\dimen143
\Gm@odd@mp=\dimen144
\Gm@even@mp=\dimen145
\Gm@layoutwidth=\dimen146
\Gm@layoutheight=\dimen147
\Gm@layouthoffset=\dimen148
\Gm@layoutvoffset=\dimen149
\Gm@dimlist=\toks18
)
(/usr/share/texmf-dist/tex/latex/listings/listings.sty
\lst@mode=\count196
\lst@gtempboxa=\box51
\lst@token=\toks19
\lst@length=\count197
\lst@currlwidth=\dimen150
\lst@column=\count198
\lst@pos=\count199
\lst@lostspace=\dimen151
\lst@width=\dimen152
\lst@newlines=\count266
\lst@lineno=\count267
\lst@maxwidth=\dimen153

(/usr/share/texmf-dist/tex/latex/listings/lstpatch.sty
File: lstpatch.sty 2024/02/21 1.10 (Carsten Heinz)
)
(/usr/share/texmf-dist/tex/latex/listings/lstmisc.sty
File: lstmisc.sty 2024/02/21 1.10 (Carsten Heinz)
\c@lstnumber=\count268
\lst@skipnumbers=\count269
\lst@framebox=\box52
)
(/usr/share/texmf-dist/tex/latex/listings/listings.cfg
File: listings.cfg 2024/02/21 1.10 listings configuration
))
Package: listings 2024/02/21 1.10 (Carsten Heinz)

(/usr/share/texmf-dist/tex/latex/xcolor/xcolor.sty
Package: xcolor 2023/11/15 v3.01 LaTeX color extensions (UK)

(/usr/share/texmf-dist/tex/latex/graphics-cfg/color.cfg
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package xcolor Info: Driver file: xetex.def on input line 274.

(/usr/share/texmf-dist/tex/latex/graphics-def/xetex.def
File: xetex.def 2022/09/22 v5.0n Graphics/color driver for xetex
)
(/usr/share/texmf-dist/tex/latex/graphics/mathcolor.ltx)
Package xcolor Info: Model `cmy' substituted by `cmy0' on input line 1350.
Package xcolor Info: Model `RGB' extended on input line 1366.
Package xcolor Info: Model `HTML' substituted by `rgb' on input line 1368.
Package xcolor Info: Model `Hsb' substituted by `hsb' on input line 1369.
Package xcolor Info: Model `tHsb' substituted by `hsb' on input line 1370.
Package xcolor Info: Model `HSB' substituted by `hsb' on input line 1371.
Package xcolor Info: Model `Gray' substituted by `gray' on input line 1372.
Package xcolor Info: Model `wave' substituted by `hsb' on input line 1373.
)
(/usr/share/texmf-dist/tex/latex/l3backend/l3backend-xetex.def
File: l3backend-xetex.def 2024-02-20 L3 backend support: XeTeX
\g__graphics_track_int=\count270
\l__pdf_internal_box=\box53
\g__pdf_backend_object_int=\count271
\g__pdf_backend_annotation_int=\count272
\g__pdf_backend_link_int=\count273
) (./main.aux)
\openout1 = `main.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 22.
LaTeX Font Info:    Trying to load font information for TS1+cmr on input line 2
2.

(/usr/share/texmf-dist/tex/latex/base/ts1cmr.fd
File: ts1cmr.fd 2023/04/13 v2.5m Standard LaTeX font definitions
)
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for TU/lmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.

*geometry* driver: auto-detecting
*geometry* detected driver: xetex
*geometry* verbose mode - [ preamble ] result:
* driver: xetex
* paper: a4paper
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: 
* h-part:(L,W,R)=(20.0pt, 557.50787pt, 20.0pt)
* v-part:(T,H,B)=(20.0pt, 805.04684pt, 20.0pt)
* \paperwidth=597.50787pt
* \paperheight=845.04684pt
* \textwidth=557.50787pt
* \textheight=805.04684pt
* \oddsidemargin=-52.26999pt
* \evensidemargin=-52.26999pt
* \topmargin=-91.26999pt
* \headheight=14.0pt
* \headsep=25.0pt
* \topskip=14.0pt
* \footskip=30.0pt
* \marginparwidth=15.0pt
* \marginparsep=10.0pt
* \columnsep=10.0pt
* \skip\footins=12.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumnfalse
* \@twosidefalse
* \@mparswitchfalse
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)

\c@lstlisting=\count274
(/usr/share/texmf-dist/tex/latex/listings/lstlang1.sty
File: lstlang1.sty 2024/02/21 1.10 listings language file
) [1

] (./main.aux)
 ***********
LaTeX2e <2023-11-01> patch level 1
L3 programming layer <2024-02-20>
 ***********
 ) 
Here is how much of TeX's memory you used:
 3532 strings out of 474773
 54820 string characters out of 5763518
 1930839 words of memory out of 5000000
 25713 multiletter control sequences out of 15000+600000
 558085 words of font info for 38 fonts, out of 8000000 for 9000
 1348 hyphenation exceptions out of 8191
 57i,5n,65p,183b,1628s stack positions out of 10000i,1000n,20000p,200000b,200000s

Output written on main.pdf (1 page).
