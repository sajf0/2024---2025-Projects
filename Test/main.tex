\documentclass[14pt, a4paper]{extreport}

\usepackage[
margin=20pt
]{geometry}

\usepackage{listings}

\usepackage{xcolor}

\lstset{
basicstyle=\ttfamily,
keywordstyle=\color{blue},
commentstyle=\color{gray},
stringstyle=\color{red},
numbers=left,
numberstyle=\color{gray},
frame=single,
breaklines=true
}

\begin{document}

\begin{center}
	\begin{lstlisting}[
	language=Matlab,
	]
	for i=1:N
		disp(i);
	end
	
	title('DQM')
\end{lstlisting}
\end{center}
	
\end{document}