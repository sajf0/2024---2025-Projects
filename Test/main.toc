\babel@toc {arabic}{}\relax 
\contentsline {section}{\numberline {1}مقدمة}{1}{}%
