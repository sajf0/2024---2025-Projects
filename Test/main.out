\BOOKMARK [0][-]{chapter*.2}{\376\377\006\105\006\102\006\057\006\105\006\051}{}% \babelsublr  {1}
\BOOKMARK [0][-]{chapter.1}{\376\377\006\106\006\070\006\061\006\051\000\040\006\071\006\047\006\105\006\051}{}% \babelsublr  {2}
\BOOKMARK [1][-]{section.1.1}{\376\377\006\110\006\054\006\107\000\040\006\047\006\104\006\052\006\063\006\105\006\112\006\051}{chapter.1}% \babelsublr  {3}
\BOOKMARK [1][-]{section.1.2}{\376\377\006\052\006\047\006\061\006\112\006\056}{chapter.1}% \babelsublr  {4}
\BOOKMARK [2][-]{subsection.1.2.1}{\376\377\006\047\006\104\006\052\006\112\006\047\006\061\000\040\006\047\006\104\006\103\006\107\006\061\006\050\006\047\006\046\006\112}{section.1.2}% \babelsublr  {5}
\BOOKMARK [2][-]{subsection.1.2.2}{\376\377\006\054\006\057\006\110\006\104\000\040\006\047\006\104\006\052\006\110\006\062\006\112\006\071\000\040\006\047\006\104\006\045\006\104\006\103\006\052\006\061\006\110\006\106\006\112}{section.1.2}% \babelsublr  {6}
